web_filter_description/static/src/xml/web_filter_description.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <div t-name="SearchView.FavoriteMenu" class="btn-group o_dropdown">
+        <button class="o_dropdown_toggler_btn btn btn-sm dropdown-toggle" data-toggle="dropdown">
```

Comment:
You can use t-extend and t-jquery here

`
<t t-extend="SearchView.FavoriteMenu">
     <t t-jquery="..." t-operation="...">
           ...
     </t>
</t>
`